packages/edit-site/src/hooks/push-changes-to-global-styles/index.js

```diff
@@ -137,7 +137,7 @@ function PushChangesToGlobalStylesControl( {
 				disabled={ changes.length === 0 }
 				onClick={ pushChanges }
 			>
-				{ __( 'Push changes to Global Styles' ) }
+				{ __( 'Apply globally' ) }
```

Comment:
Looking great!

We may also want to update the description (line #27) as [discussed here](https://github.com/WordPress/gutenberg/issues/46928#issuecomment-1375557951)